velox/common/base/SimdUtil-inl.h

```diff
@@ -294,7 +294,8 @@ inline bool copyNextWord(void*& to, const void*& from, int32_t& bytes) {
 } // namespace detail
 
 template <typename A>
-void memcpy(void* to, const void* from, int32_t bytes, const A& arch) {
+FOLLY_ALWAYS_INLINE void
```

Comment:
Did you notice big difference for inlining this?  This function seems too large to benefit from inlining.